scripts/release.js

```diff
@@ -0,0 +1,35 @@
+const path = require("path");
+const { exec, getExecOutput } = require("@actions/exec");
+const tag = `v${require("../package.json").version}`;
```

Comment:
There is also a question - what version should we release first? I probably would say that we should immediately bump to v1 - leaving a possibility to add all missing tags retroactively in the future